src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -16,6 +21,7 @@ public class CookieRequestCultureProvider : RequestCultureProvider
         private static readonly char[] _cookieSeparator = new[] { '|' };
         private static readonly string _culturePrefix = "c=";
         private static readonly string _uiCulturePrefix = "uic=";
+        private static ILogger _logger;
```

Comment:
Sorry, I just had a look at the type in entirety. We'd have to pass the logger explicitly in to a separate overload. Essentially we can have it log when `DetermineProviderCultureResult` is invoked using the new ctor.